apps/src/applab/designElements/DeleteElementButton.jsx

```diff
@@ -37,7 +38,8 @@ var DeleteElementButton = React.createClass({
     var buttonStyle = {
       paddingTop: '5px',
       paddingBottom: '5px',
-      fontSize: '14px'
+      fontSize: '14px',
+      marginLeft: 18
```

Comment:
Do we want this margin when in the confirmation portion? I would think that would result in big gaps between the buttons. I'd also make sure that we end up with a case where I have to move the mouse between clicking on delete and clicking on yes.